src/SeleniumLibrary/keywords/formelement.py

```diff
@@ -192,72 +193,78 @@ def select_radio_button(self, group_name, value):
 
     @keyword
     def choose_file(self, locator, file_path):
-        """Inputs the `file_path` into file input field found by `locator`.
+        """Inputs the ``file_path`` into file input field ``locator``.
 
         This keyword is most often used to input files into upload forms.
-        The file specified with `file_path` must be available on the same host
-        where the Selenium Server is running.
+        The file specified with ``file_path`` must be available on machine
+        where tests are executed.
 
         Example:
-        | Choose File | my_upload_field | /home/user/files/trades.csv |
+        | `Choose File` | my_upload_field | ${CURDIR}/trades.csv |
         """
         if not os.path.isfile(file_path):
             raise ValueError("File '%s' does not exist on the local file "
                              "system." % file_path)
         self.find_element(locator).send_keys(file_path)
 
     @keyword
-    def input_password(self, locator, text):
-        """Types the given password into text field identified by `locator`.
+    def input_password(self, locator, password):
+        """Types the given password into text field identified by ``locator``.
 
         Difference between this keyword and `Input Text` is that this keyword
-        does not log the given password. See `introduction` for details about
-        locating elements.
+        does not log the given password.
```

Comment:
This is not totally true, the password is logged if loglevel is high enough to display selenium debug messages. See issue  https://github.com/robotframework/SeleniumLibrary/issues/202 for more details.

Reply:
True. I didn't actually remember that SeleniumLibrary logs this on DEBUG level. Need to make it more explicit that password isn't logged on INFO level.